src/Tizen.NUI.Components/Controls/Navigation/DialogPage.cs

```diff
@@ -284,5 +284,65 @@ public static void ShowAlertDialog(string title, string message, params View[] a
 
             NUIApplication.GetDefaultWindow().GetDefaultNavigator().Push(dialogPage);
         }
+
+        /// <summary>
+        /// Shows a menu by pushing a dialog page containing menu to default navigator.
+        /// </summary>
+        /// <param name="anchor">The anchor view where menu is displayed.</param>
+        /// <param name="items">The menu items.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [SuppressMessage("Microsoft.Reliability",
+                         "CA2000:DisposeObjectsBeforeLosingScope",
+                         Justification = "The pushed views are added to NavigationPages and are disposed in Navigator.Dispose().")]
+        public static void ShowMenu(View anchor, params MenuItem[] items)
+        {
+            if (items == null)
+            {
+                return;
+            }
+
+            Position2D anchorPosition = new Position2D((int)(anchor?.ScreenPosition.X ?? 0), (int)(anchor?.ScreenPosition.Y ?? 0) + (anchor?.Size2D.Height ?? 0) + (anchor?.Margin.Bottom ?? 0));
+
+            var dialogPage = new DialogPage()
+            {
+                Content = new Menu()
+                {
+                    Items = items,
+                    AnchorPosition = anchorPosition,
+                },
+                ScrimColor = Color.Transparent,
+            };
+
+            NUIApplication.GetDefaultWindow().GetDefaultNavigator().Push(dialogPage);
```

Comment:
Isn't it necessary in a multi-window situation?

Reply:
Thank you for your comment.
I think it's good point.
For now, static helper methods only supports default window and default navigator for simple form of method.
I think we can overload this method with window and navigator parameters when we meet developers' requests.
```
pubic static void ShowMenu(View anchor, params MenuItem[] items, Window window, Navigator navigator)

Reply:
Ok